apps/DemoApp/DemoApp/PlayerViewController.swift

```diff
@@ -500,3 +552,27 @@ extension PlayerViewController {
         self.view.addGestureRecognizer(swipeLeftRecognizer)
     }
 }
+
+extension PlayerViewController: AVPictureInPictureControllerDelegate {
+    
+    func pictureInPictureController(_ pictureInPictureController: AVPictureInPictureController,
+                                    restoreUserInterfaceForPictureInPictureStopWithCompletionHandler completionHandler: @escaping (Bool) -> Void) {
+        // Restore user interface
+        completionHandler(true)
+    }
+    
+    func pictureInPictureControllerWillStartPictureInPicture(_ pictureInPictureController: AVPictureInPictureController) {
+        // hide playback controls
+        // show placeholder artwork
```

Comment:
what is the placeholder artwork? I am a little bit confused why do we have two rowStacks that we have to hide and show instead of a single one with the airplayButton.

Reply:
This artwork is the background screen when PiP is active. And the row rows I'm hidding is the one with the airplayButton and the other one is the one with the main actions buttons (play/pause and the slider) Should I just hide the one with the airplayButton?
![C00C483D-9EA0-4D6E-BA21-5F050870FF15](https://user-images.githubusercontent.com/94012251/155554388-f2ed7b68-a44a-42b2-97a9-ef171fd7ae68.png)

Reply:
are you able to use the main actions (play/pause/slider) when in PiP? If you can, then I think it makes sense to show it, if you can't then we should hide it

Reply:
Yes we can use the main actions when in PiP, so I'll just remove that line